.gitlab/internal_image_deploy/internal_image_deploy.yml

```
@@ -47,6 +47,51 @@ docker_trigger_internal:
       --variable TARGET_ENV
       --variable DYNAMIC_BUILD_RENDER_TARGET_FORWARD_PARAMETERS"
 
+docker_trigger_internal-fips:
+  stage: internal_image_deploy
+  rules: !reference [.on_deploy_internal_or_manual]
+  needs:
+    - job: docker_build_fips_agent7
+      artifacts: false
+    - job: docker_build_fips_agent7_arm64
+      artifacts: false
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
```

```suggestion
  image: registry.ddbuild.io/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
```

Now, build images are located to registry.ddbuild.io

---

fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/4b28e0761ae698629c3f7a21df38fb883e29df51